tests/test_oup_affiliation_parsing.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from hepcrawl.extractors import oup_parser
+import os
+from scrapy.selector import Selector
+
+files_for_testing = ['2022_oup_ptac032.xml', '2021_oup_ptab168.xml', '2020_oup_ptaa186.xml']
+correct_affiliations = {files_for_testing[0]:
+                        ["Department of Physics, Graduate School of Science, Osaka University, , , Toyonaka, Osaka 560-0043, , , Japan"],
+                        files_for_testing[1]: [
+                            "Center for Gravitational Physics, Yukawa Institute for Theoretical Physics, Kyoto University, , , Kyoto 606-8502, , , Japan"],
+                        files_for_testing[2]: [
+                            "Institute of Science and Engineering, , Shimane University, , Matsue 690-8504, , Japan"]}
+
+
+@pytest.fixture
+def affiliations_from_records():
+    parsed_affiliations = {}
+
+    cwd = os.getcwd()
+    files = [os.path.join(cwd, 'responses', 'oup', file_name) for file_name in files_for_testing]
+    for file in files:
+        parser = oup_parser.OUPParser()
+        with open(os.path.join(os.getcwd(), 'responses', file), 'r') as file_:
+            selector = Selector(text=file_.read(), type='xml')
+            affiliations = parser._get_authors(selector)
+            parsed_affiliations[os.path.basename(file)] = affiliations
+    assert parsed_affiliations
+    return parsed_affiliations
```

Comment:
`pytest-datadir`, provides a better way to read files https://github.com/gabrielcnr/pytest-datadir